src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1549,7 +1549,7 @@ public function getLanguages()
         foreach (array_keys($languages) as $lang) {
```

Comment:
It's not so much work (I recall 10-15 warnings from analyzer), but side-effect is unused variables.

I'm not sure, so it worth discussing with maintainers internally and mentioning me with final decision.